### PR TITLE
Feat/add tag filter

### PR DESCRIPTION
**Pull request by sumanjha108155**
## Description

The UI has been updated and made responsive. The tags were missing `any` option which has been modified. 

---

## Issue

- [x] Partially fixes - #36

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ const App = () => {
         );
 
         setQuotes(data.data);
+        setOfTags.add("any")
```
Comment (nirajgiriXD) [inline src/App.tsx]:
```suggestion
        setOfTags.add("any");
```

File: src/assets/css/style.css [hunk 2]
```diff
@@ -46,31 +46,13 @@ nav {
...
-}
-
-.quote-author {
-  height: auto;
-  clear: both;
-  font-size: 1em;
-  text-align: right;
-  margin-bottom: 5em;
-}
-
```
Comment (nirajgiriXD) [inline src/assets/css/style.css:67]: This shouldn't have been deleted.

File: src/components/quoteBox/quoteBox.tsx [hunk 3]
```diff
@@ -47,68 +49,58 @@ const QuoteBox: React.FC<QuoteBoxProp> = ({ quotes, tags }) => {
...
-                        <Dropdown.Item eventKey={tag} key={tag}>
-                          {tag[0].toUpperCase() + tag.substring(1)}
-                        </Dropdown.Item>
-                      );
-                    })}
-                  </Dropdown.Menu>
-                </Dropdown>
-              </div>
+        <div className="display-container p-5">
+            <div className="row border-bottom border-dark border-2">
```
Comment (nirajgiriXD) [inline src/components/quoteBox/quoteBox.tsx]:
```suggestion
            <div className="row border-bottom">
```
Light border is preferred over heavy border, in this case.

File: src/assets/css/style.css (hunk 2 above)
Comment (sumanjha108155) [reply src/assets/css/style.css:67]: The code made the content of the quotes unresponsive.  I used respective bootstrap classes that performs same task but are responsive in nature.

**Review by nirajgiriXD — COMMENTED** [review]
The new commits which aims aims to resolve the issue has already been resolved.

**Comment by nirajgiriXD** [comment]
Closing the PR as the issues has been resolved and merged into the `main` branch.

File: src/assets/css/style.css (hunk 2 above)
Comment (nirajgiriXD) [reply src/assets/css/style.css:67]: Hmm, ok. But the general approach should have been to wrap the element with another element and apply CSS classes to that wrapper element. By doing so, we won't have to remove the custom CSS, also at the same time responsiveness of the component would have been achieved.

Comment (sumanjha108155) [reply src/assets/css/style.css:67]: Sure will keep that in mind.

**Pull request closed without merging**